Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/QuotasDbContext.cs

```diff
@@ -20,6 +20,8 @@ public class QuotasDbContext : AbstractDbContextBase
 
     public DbSet<TierQuota> TierQuotas { get; set; }
 
+    public DbSet<IndividualQuota> IndividualQuotas { get; set; }
```

Comment:
We shouldn't need this. Also, the `TierQuotas` and `TierQuotasDefinitions` aren't currently used, so they can be removed as well.

To make sure it doesn't affect the migrations, try to create one. If there are changes, tell me. But I don't expect there to be.